flows/vision-zero/cris_import/lib/sql.py

```diff
@@ -439,8 +439,8 @@ def show_changed_values(
         print(column)
         linkage_sql = " and ".join(linkage_clauses)
         sql = f"""
-            select import.{table}.{column}::text as import_{column},
-                   public.{output_map[table]}.{column}::text as public_{column}
+            select {DB_IMPORT_SCHEMA}.{table}.{column}::text as import_{column},
+                public.{output_map[table]}.{column}::text as public_{column}
```

Comment:
I really thought I could get by without editing this file. 🫰🏻

This is a bug where I had hard-coded `import` into a SQL statement when I really wanted to interpolate the passed-in value `DB_IMPORT_SCHEMA`.